src/components/checkbox/checkbox.template.tsx

```diff
@@ -14,6 +15,11 @@ interface CheckboxTemplateProps {
 export const CheckboxTemplate: FunctionalComponent<CheckboxTemplateProps> = (
     props
 ) => {
+    const indeterminateProps = {};
```

Comment:
I think it makes more sense that this would be called `inputProps` 🤔

Reply:
For sure! Good suggestion, I'll rename it.